From 25d96119b45eca63401d2d5d9bb86629b9596810 Mon Sep 17 00:00:00 2001
From: Pino Toscano <pino@kde.org>
Date: Sat, 18 Aug 2018 17:37:22 +0200
Subject: [PATCH 08/35] no-op open&save .ui files

Open & save all the .ui files, making sure to not change the shortcuts
(even if they comflict).  This way, further changes will produce less
differences, and thus they will be easier to read/review.
---
 src/dialogs/floatdialog_base.ui             | 125 ++++++++++--------
 src/dialogs/includegraphicsdialog_base.ui   |   6 +-
 src/dialogs/newtoolwizard_class_page.ui     |  41 +++---
 src/dialogs/newtoolwizard_toolname_page.ui  |  47 +++----
 src/dialogs/pdf-wizard/pdfdialog_base.ui    |  15 ++-
 src/dialogs/postscriptdialog_base.ui        | 133 ++++++++++----------
 src/dialogs/scriptshortcutdialog_base.ui    |   2 +-
 src/dialogs/tabbingdialog_base.ui           |  58 ++++-----
 src/dialogs/usermenu/usermenudialog_base.ui |  16 +--
 src/widgets/configcheckerwidget.ui          |  50 ++++----
 src/widgets/graphicsconfigwidget.ui         |   2 +-
 src/widgets/latexconfigwidget.ui            |  12 +-
 src/widgets/latextoolconfigwidget.ui        |  47 ++++---
 src/widgets/maintoolconfigwidget.ui         |  16 ++-
 src/widgets/newdocumentwidget.ui            |  54 ++++----
 src/widgets/processtoolconfigwidget.ui      |   1 -
 src/widgets/quicktoolconfigwidget.ui        |  13 +-
 src/widgets/scriptingconfigwidget.ui        | 104 +++++++--------
 src/widgets/structureviewconfigwidget.ui    |   2 +-
 src/widgets/symbolviewconfigwidget.ui       |   2 +-
 20 files changed, 390 insertions(+), 356 deletions(-)

diff --git a/src/dialogs/floatdialog_base.ui b/src/dialogs/floatdialog_base.ui
index ccab84bb..6992a37b 100644
--- a/src/dialogs/floatdialog_base.ui
+++ b/src/dialogs/floatdialog_base.ui
@@ -1,7 +1,8 @@
-<ui version="4.0" >
+<?xml version="1.0" encoding="UTF-8"?>
+<ui version="4.0">
  <class>FloatDialog</class>
- <widget class="QWidget" name="FloatDialog" >
-  <property name="geometry" >
+ <widget class="QWidget" name="FloatDialog">
+  <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
@@ -9,29 +10,38 @@
     <height>351</height>
    </rect>
   </property>
-  <layout class="QVBoxLayout" >
-   <property name="margin" >
+  <layout class="QVBoxLayout">
+   <property name="leftMargin">
+    <number>0</number>
+   </property>
+   <property name="topMargin">
+    <number>0</number>
+   </property>
+   <property name="rightMargin">
+    <number>0</number>
+   </property>
+   <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
-    <widget class="QGroupBox" name="groupBox" >
-     <property name="title" >
+    <widget class="QGroupBox" name="groupBox">
+     <property name="title">
       <string>Environment</string>
      </property>
-     <layout class="QHBoxLayout" >
+     <layout class="QHBoxLayout">
       <item>
-       <widget class="QRadioButton" name="m_rbFigure" >
-        <property name="text" >
+       <widget class="QRadioButton" name="m_rbFigure">
+        <property name="text">
          <string>Figure</string>
         </property>
-        <property name="checked" >
+        <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
-       <widget class="QRadioButton" name="m_rbTable" >
-        <property name="text" >
+       <widget class="QRadioButton" name="m_rbTable">
+        <property name="text">
          <string>Table</string>
         </property>
        </widget>
@@ -40,44 +50,44 @@
     </widget>
    </item>
    <item>
-    <widget class="QGroupBox" name="groupBox_2" >
-     <property name="title" >
+    <widget class="QGroupBox" name="groupBox_2">
+     <property name="title">
       <string>Position</string>
      </property>
-     <layout class="QGridLayout" >
-      <item row="0" column="0" >
-       <widget class="QCheckBox" name="m_cbHere" >
-        <property name="text" >
+     <layout class="QGridLayout">
+      <item row="0" column="0">
+       <widget class="QCheckBox" name="m_cbHere">
+        <property name="text">
          <string>Here exact</string>
         </property>
-        <property name="checked" >
+        <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
-      <item row="0" column="1" >
-       <widget class="QCheckBox" name="m_cbBottom" >
-        <property name="text" >
+      <item row="0" column="1">
+       <widget class="QCheckBox" name="m_cbBottom">
+        <property name="text">
          <string>Bottom of page</string>
         </property>
        </widget>
       </item>
-      <item row="1" column="0" >
-       <widget class="QCheckBox" name="m_cbTop" >
-        <property name="text" >
+      <item row="1" column="0">
+       <widget class="QCheckBox" name="m_cbTop">
+        <property name="text">
          <string>Top of page</string>
         </property>
-        <property name="checked" >
+        <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
-      <item row="1" column="1" >
-       <widget class="QCheckBox" name="m_cbPage" >
-        <property name="text" >
+      <item row="1" column="1">
+       <widget class="QCheckBox" name="m_cbPage">
+        <property name="text">
          <string>Extra page</string>
         </property>
-        <property name="checked" >
+        <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
@@ -86,53 +96,62 @@
     </widget>
    </item>
    <item>
-    <widget class="QWidget" native="1" name="widget" >
-     <layout class="QGridLayout" >
-      <property name="margin" >
+    <widget class="QWidget" name="widget" native="true">
+     <layout class="QGridLayout">
+      <property name="leftMargin">
+       <number>0</number>
+      </property>
+      <property name="topMargin">
+       <number>0</number>
+      </property>
+      <property name="rightMargin">
+       <number>0</number>
+      </property>
+      <property name="bottomMargin">
        <number>0</number>
       </property>
-      <item row="0" column="1" >
-       <widget class="QCheckBox" name="m_cbCenter" >
-        <property name="text" >
+      <item row="0" column="1">
+       <widget class="QCheckBox" name="m_cbCenter">
+        <property name="text">
          <string>Center</string>
         </property>
-        <property name="checked" >
+        <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
-      <item row="1" column="0" >
-       <widget class="QLabel" name="label_2" >
-        <property name="text" >
+      <item row="1" column="0">
+       <widget class="QLabel" name="label_2">
+        <property name="text">
          <string>Caption:</string>
         </property>
-        <property name="buddy" >
+        <property name="buddy">
          <cstring>m_edCaption</cstring>
         </property>
        </widget>
       </item>
-      <item row="1" column="1" >
-       <widget class="KLineEdit" name="m_edCaption" />
+      <item row="1" column="1">
+       <widget class="KLineEdit" name="m_edCaption"/>
       </item>
-      <item row="2" column="0" >
-       <widget class="QLabel" name="label_3" >
-        <property name="text" >
+      <item row="2" column="0">
+       <widget class="QLabel" name="label_3">
+        <property name="text">
          <string>Label:</string>
         </property>
-        <property name="buddy" >
+        <property name="buddy">
          <cstring>m_edLabel</cstring>
         </property>
        </widget>
       </item>
-      <item row="2" column="1" >
-       <widget class="KLineEdit" name="m_edLabel" />
+      <item row="2" column="1">
+       <widget class="KLineEdit" name="m_edLabel"/>
       </item>
-      <item row="3" column="1" >
+      <item row="3" column="1">
        <spacer>
-        <property name="orientation" >
+        <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
-        <property name="sizeHint" stdset="0" >
+        <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
diff --git a/src/dialogs/includegraphicsdialog_base.ui b/src/dialogs/includegraphicsdialog_base.ui
index 05bb52da..acc9a88d 100644
--- a/src/dialogs/includegraphicsdialog_base.ui
+++ b/src/dialogs/includegraphicsdialog_base.ui
@@ -6,8 +6,8 @@
    <rect>
     <x>0</x>
     <y>0</y>
-    <width>469</width>
-    <height>588</height>
+    <width>485</width>
+    <height>629</height>
    </rect>
   </property>
   <layout class="QVBoxLayout">
@@ -542,7 +542,7 @@
  <customwidgets>
   <customwidget>
    <class>KUrlRequester</class>
-   <extends>QFrame</extends>
+   <extends>QWidget</extends>
    <header>kurlrequester.h</header>
   </customwidget>
  </customwidgets>
diff --git a/src/dialogs/newtoolwizard_class_page.ui b/src/dialogs/newtoolwizard_class_page.ui
index 74e0e8bc..2d28015a 100644
--- a/src/dialogs/newtoolwizard_class_page.ui
+++ b/src/dialogs/newtoolwizard_class_page.ui
@@ -1,7 +1,8 @@
-<ui version="4.0" >
+<?xml version="1.0" encoding="UTF-8"?>
+<ui version="4.0">
  <class>NewToolWizardClassPage</class>
- <widget class="QWidget" name="NewToolWizardClassPage" >
-  <property name="geometry" >
+ <widget class="QWidget" name="NewToolWizardClassPage">
+  <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
@@ -9,34 +10,34 @@
     <height>209</height>
    </rect>
   </property>
-  <layout class="QHBoxLayout" >
+  <layout class="QHBoxLayout">
    <item>
-    <layout class="QVBoxLayout" >
+    <layout class="QVBoxLayout">
      <item>
-      <widget class="QLabel" name="m_lbBehavior" >
-       <property name="text" >
+      <widget class="QLabel" name="m_lbBehavior">
+       <property name="text">
         <string>Select the default &amp;behavior (class)
 of this tool. It will inherit all properties
 of the tool it is based upon.
 
-For example, selecting "LaTeX" will
+For example, selecting &quot;LaTeX&quot; will
 cause your tool to behave just like
-the standard "LaTeX" tool.</string>
+the standard &quot;LaTeX&quot; tool.</string>
        </property>
-       <property name="wordWrap" >
+       <property name="wordWrap">
         <bool>false</bool>
        </property>
-       <property name="buddy" >
+       <property name="buddy">
         <cstring>m_cbTools</cstring>
        </property>
       </widget>
      </item>
      <item>
       <spacer>
-       <property name="orientation" >
+       <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
-       <property name="sizeHint" >
+       <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
@@ -48,10 +49,10 @@ the standard "LaTeX" tool.</string>
    </item>
    <item>
     <spacer>
-     <property name="orientation" >
+     <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
-     <property name="sizeHint" >
+     <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
@@ -60,19 +61,19 @@ the standard "LaTeX" tool.</string>
     </spacer>
    </item>
    <item>
-    <layout class="QVBoxLayout" >
+    <layout class="QVBoxLayout">
      <item>
-      <widget class="KComboBox" name="m_cbTools" />
+      <widget class="KComboBox" name="m_cbTools"/>
      </item>
      <item>
       <spacer>
-       <property name="orientation" >
+       <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
-       <property name="sizeType" >
+       <property name="sizeType">
         <enum>QSizePolicy::Expanding</enum>
        </property>
-       <property name="sizeHint" >
+       <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>111</height>
diff --git a/src/dialogs/newtoolwizard_toolname_page.ui b/src/dialogs/newtoolwizard_toolname_page.ui
index 579de8d2..1132a3b3 100644
--- a/src/dialogs/newtoolwizard_toolname_page.ui
+++ b/src/dialogs/newtoolwizard_toolname_page.ui
@@ -1,7 +1,8 @@
-<ui version="4.0" >
+<?xml version="1.0" encoding="UTF-8"?>
+<ui version="4.0">
  <class>NewToolWizardToolNamePage</class>
- <widget class="QWidget" name="NewToolWizardToolNamePage" >
-  <property name="geometry" >
+ <widget class="QWidget" name="NewToolWizardToolNamePage">
+  <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
@@ -9,28 +10,28 @@
     <height>248</height>
    </rect>
   </property>
-  <layout class="QVBoxLayout" >
+  <layout class="QVBoxLayout">
    <item>
-    <layout class="QHBoxLayout" >
+    <layout class="QHBoxLayout">
      <item>
-      <widget class="QLabel" name="m_lbName" >
-       <property name="text" >
+      <widget class="QLabel" name="m_lbName">
+       <property name="text">
         <string>Type a short descriptive name for the &amp;tool:</string>
        </property>
-       <property name="wordWrap" >
+       <property name="wordWrap">
         <bool>false</bool>
        </property>
-       <property name="buddy" >
+       <property name="buddy">
         <cstring>m_leName</cstring>
        </property>
       </widget>
      </item>
      <item>
       <spacer>
-       <property name="orientation" >
+       <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
-       <property name="sizeHint" >
+       <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
@@ -39,20 +40,20 @@
       </spacer>
      </item>
      <item>
-      <widget class="KLineEdit" name="m_leName" >
-       <property name="sizePolicy" >
-        <sizepolicy vsizetype="Fixed" hsizetype="MinimumExpanding" >
+      <widget class="KLineEdit" name="m_leName">
+       <property name="sizePolicy">
+        <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
-       <property name="minimumSize" >
+       <property name="minimumSize">
         <size>
          <width>100</width>
          <height>0</height>
         </size>
        </property>
-       <property name="text" >
+       <property name="text">
         <string/>
        </property>
       </widget>
@@ -60,16 +61,16 @@
     </layout>
    </item>
    <item>
-    <layout class="QVBoxLayout" >
+    <layout class="QVBoxLayout">
      <item>
       <spacer>
-       <property name="orientation" >
+       <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
-       <property name="sizeType" >
+       <property name="sizeType">
         <enum>QSizePolicy::Expanding</enum>
        </property>
-       <property name="sizeHint" >
+       <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>70</height>
@@ -78,11 +79,11 @@
       </spacer>
      </item>
      <item>
-      <widget class="QLabel" name="m_lbWarning" >
-       <property name="text" >
+      <widget class="QLabel" name="m_lbWarning">
+       <property name="text">
         <string/>
        </property>
-       <property name="wordWrap" >
+       <property name="wordWrap">
         <bool>false</bool>
        </property>
       </widget>
diff --git a/src/dialogs/pdf-wizard/pdfdialog_base.ui b/src/dialogs/pdf-wizard/pdfdialog_base.ui
index 055dfe33..99810ffe 100644
--- a/src/dialogs/pdf-wizard/pdfdialog_base.ui
+++ b/src/dialogs/pdf-wizard/pdfdialog_base.ui
@@ -7,7 +7,7 @@
     <x>0</x>
     <y>0</y>
     <width>562</width>
-    <height>550</height>
+    <height>563</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_4">
@@ -592,7 +592,16 @@
              <property name="spacing">
               <number>0</number>
              </property>
-             <property name="margin">
+             <property name="leftMargin">
+              <number>0</number>
+             </property>
+             <property name="topMargin">
+              <number>0</number>
+             </property>
+             <property name="rightMargin">
+              <number>0</number>
+             </property>
+             <property name="bottomMargin">
               <number>0</number>
              </property>
             </layout>
@@ -844,7 +853,7 @@
   </customwidget>
   <customwidget>
    <class>KUrlRequester</class>
-   <extends>QFrame</extends>
+   <extends>QWidget</extends>
    <header>kurlrequester.h</header>
   </customwidget>
  </customwidgets>
diff --git a/src/dialogs/postscriptdialog_base.ui b/src/dialogs/postscriptdialog_base.ui
index 58ba8812..5ec0e024 100644
--- a/src/dialogs/postscriptdialog_base.ui
+++ b/src/dialogs/postscriptdialog_base.ui
@@ -1,7 +1,8 @@
-<ui version="4.0" >
+<?xml version="1.0" encoding="UTF-8"?>
+<ui version="4.0">
  <class>PostscriptDialog</class>
- <widget class="QWidget" name="PostscriptDialog" >
-  <property name="geometry" >
+ <widget class="QWidget" name="PostscriptDialog">
+  <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
@@ -9,143 +10,143 @@
     <height>377</height>
    </rect>
   </property>
-  <layout class="QVBoxLayout" >
-   <property name="leftMargin" >
+  <layout class="QVBoxLayout">
+   <property name="leftMargin">
     <number>0</number>
    </property>
-   <property name="topMargin" >
+   <property name="topMargin">
     <number>0</number>
    </property>
-   <property name="rightMargin" >
+   <property name="rightMargin">
     <number>0</number>
    </property>
-   <property name="bottomMargin" >
+   <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
-    <widget class="QGroupBox" name="groupBox" >
-     <property name="title" >
+    <widget class="QGroupBox" name="groupBox">
+     <property name="title">
       <string>Parameter</string>
      </property>
-     <layout class="QGridLayout" >
-      <item row="0" column="0" colspan="2" >
-       <widget class="QLabel" name="m_lbInfo" >
-        <property name="text" >
+     <layout class="QGridLayout">
+      <item row="0" column="0" colspan="2">
+       <widget class="QLabel" name="m_lbInfo">
+        <property name="text">
          <string>Conversion of ps files is made by 'pstops' and 'psselect'.
 Be sure to call 'dvips' with option '-t a4' and
 hyperref package (if needed) with option 'a4paper'.</string>
         </property>
-        <property name="alignment" >
+        <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
-      <item row="1" column="0" >
-       <widget class="QLabel" name="label_2" >
-        <property name="text" >
+      <item row="1" column="0">
+       <widget class="QLabel" name="label_2">
+        <property name="text">
          <string>Input file:</string>
         </property>
        </widget>
       </item>
-      <item row="1" column="1" >
-       <widget class="KUrlRequester" name="m_edInfile" >
-        <property name="whatsThis" >
+      <item row="1" column="1">
+       <widget class="KUrlRequester" name="m_edInfile">
+        <property name="whatsThis">
          <string>Input file, which should be converted.</string>
         </property>
-        <property name="filter" >
+        <property name="filter">
          <string>*.ps|PS Files\n*.ps.gz|Zipped PS Files</string>
         </property>
        </widget>
       </item>
-      <item row="2" column="0" >
-       <widget class="QLabel" name="label_3" >
-        <property name="text" >
+      <item row="2" column="0">
+       <widget class="QLabel" name="label_3">
+        <property name="text">
          <string>Output file:</string>
         </property>
        </widget>
       </item>
-      <item row="2" column="1" >
-       <widget class="KUrlRequester" name="m_edOutfile" >
-        <property name="whatsThis" >
+      <item row="2" column="1">
+       <widget class="KUrlRequester" name="m_edOutfile">
+        <property name="whatsThis">
          <string>The name of the output file. This entry may also be empty, if you only want to view the result without saving it. In this case the viewer checkbox must be checked.</string>
         </property>
-        <property name="filter" >
+        <property name="filter">
          <string>*.ps|PS Files\n*.ps.gz|Zipped PS Files</string>
         </property>
        </widget>
       </item>
-      <item row="3" column="0" >
-       <widget class="QLabel" name="label_4" >
-        <property name="text" >
+      <item row="3" column="0">
+       <widget class="QLabel" name="label_4">
+        <property name="text">
          <string>Task:</string>
         </property>
        </widget>
       </item>
-      <item row="3" column="1" >
-       <widget class="KComboBox" name="m_cbTask" >
-        <property name="whatsThis" >
+      <item row="3" column="1">
+       <widget class="KComboBox" name="m_cbTask">
+        <property name="whatsThis">
          <string>Choose one of the 18 operations to convert a postscript file. The last four operations need specific parameters.</string>
         </property>
        </widget>
       </item>
-      <item row="4" column="0" >
-       <widget class="QLabel" name="m_lbParameter" >
-        <property name="text" >
+      <item row="4" column="0">
+       <widget class="QLabel" name="m_lbParameter">
+        <property name="text">
          <string>Parameter:</string>
         </property>
        </widget>
       </item>
-      <item row="5" column="0" >
-       <widget class="QLabel" name="label_6" >
-        <property name="text" >
+      <item row="5" column="0">
+       <widget class="QLabel" name="label_6">
+        <property name="text">
          <string>Viewer:</string>
         </property>
        </widget>
       </item>
-      <item row="5" column="1" >
-       <widget class="QCheckBox" name="m_cbView" >
-        <property name="whatsThis" >
+      <item row="5" column="1">
+       <widget class="QCheckBox" name="m_cbView">
+        <property name="whatsThis">
          <string>View the result of the conversion process. okular is always taken as an external viewer.</string>
         </property>
-        <property name="text" >
+        <property name="text">
          <string>Show ps file with 'okular'</string>
         </property>
-        <property name="checked" >
+        <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
-      <item row="4" column="1" >
-       <widget class="QWidget" native="1" name="widget" >
-        <layout class="QVBoxLayout" >
-         <property name="spacing" >
+      <item row="4" column="1">
+       <widget class="QWidget" name="widget" native="true">
+        <layout class="QVBoxLayout">
+         <property name="spacing">
           <number>0</number>
          </property>
-         <property name="leftMargin" >
+         <property name="leftMargin">
           <number>0</number>
          </property>
-         <property name="topMargin" >
+         <property name="topMargin">
           <number>0</number>
          </property>
-         <property name="rightMargin" >
+         <property name="rightMargin">
           <number>0</number>
          </property>
-         <property name="bottomMargin" >
+         <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
-          <widget class="QLineEdit" name="m_edParameter" >
-           <property name="whatsThis" >
+          <widget class="QLineEdit" name="m_edParameter">
+           <property name="whatsThis">
             <string>'Select pages' and 'Free Parameter' need some specific parameter, which you can enter here</string>
            </property>
           </widget>
          </item>
          <item>
-          <widget class="QSpinBox" name="m_spCopies" >
-           <property name="whatsThis" >
+          <widget class="QSpinBox" name="m_spCopies">
+           <property name="whatsThis">
             <string>When you want to copy pages, you must enter the number of copies</string>
            </property>
-           <property name="minimum" >
+           <property name="minimum">
             <number>1</number>
            </property>
           </widget>
@@ -158,10 +159,10 @@ hyperref package (if needed) with option 'a4paper'.</string>
    </item>
    <item>
     <spacer>
-     <property name="orientation" >
+     <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
-     <property name="sizeHint" >
+     <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
@@ -172,16 +173,16 @@ hyperref package (if needed) with option 'a4paper'.</string>
   </layout>
  </widget>
  <customwidgets>
+  <customwidget>
+   <class>KUrlRequester</class>
+   <extends>QWidget</extends>
+   <header>kurlrequester.h</header>
+  </customwidget>
   <customwidget>
    <class>KComboBox</class>
    <extends>QComboBox</extends>
    <header>kcombobox.h</header>
   </customwidget>
-  <customwidget>
-   <class>KUrlRequester</class>
-   <extends>QFrame</extends>
-   <header>kurlrequester.h</header>
-  </customwidget>
  </customwidgets>
  <resources/>
  <connections/>
diff --git a/src/dialogs/scriptshortcutdialog_base.ui b/src/dialogs/scriptshortcutdialog_base.ui
index ff680024..43141084 100644
--- a/src/dialogs/scriptshortcutdialog_base.ui
+++ b/src/dialogs/scriptshortcutdialog_base.ui
@@ -7,7 +7,7 @@
     <x>0</x>
     <y>0</y>
     <width>302</width>
-    <height>239</height>
+    <height>275</height>
    </rect>
   </property>
   <property name="sizePolicy">
diff --git a/src/dialogs/tabbingdialog_base.ui b/src/dialogs/tabbingdialog_base.ui
index c4a6474b..0e6fd68b 100644
--- a/src/dialogs/tabbingdialog_base.ui
+++ b/src/dialogs/tabbingdialog_base.ui
@@ -1,7 +1,8 @@
-<ui version="4.0" >
+<?xml version="1.0" encoding="UTF-8"?>
+<ui version="4.0">
  <class>TabbingDialog</class>
- <widget class="QWidget" name="TabbingDialog" >
-  <property name="geometry" >
+ <widget class="QWidget" name="TabbingDialog">
+  <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
@@ -9,69 +10,68 @@
     <height>99</height>
    </rect>
   </property>
-  <layout class="QGridLayout" >
-   <property name="leftMargin" >
+  <layout class="QGridLayout">
+   <property name="leftMargin">
     <number>0</number>
    </property>
-   <property name="topMargin" >
+   <property name="topMargin">
     <number>0</number>
    </property>
-   <property name="rightMargin" >
+   <property name="rightMargin">
     <number>0</number>
    </property>
-   <property name="bottomMargin" >
+   <property name="bottomMargin">
     <number>0</number>
    </property>
-   <item row="0" column="0" >
-    <widget class="QLabel" name="label" >
-     <property name="text" >
+   <item row="0" column="0">
+    <widget class="QLabel" name="label">
+     <property name="text">
       <string>Number of &amp;columns:</string>
      </property>
-     <property name="buddy" >
+     <property name="buddy">
       <cstring>m_spCols</cstring>
      </property>
     </widget>
    </item>
-   <item row="0" column="1" >
-    <widget class="QSpinBox" name="m_spCols" >
-     <property name="minimum" >
+   <item row="0" column="1">
+    <widget class="QSpinBox" name="m_spCols">
+     <property name="minimum">
       <number>2</number>
      </property>
     </widget>
    </item>
-   <item row="1" column="0" >
-    <widget class="QLabel" name="label_2" >
-     <property name="text" >
+   <item row="1" column="0">
+    <widget class="QLabel" name="label_2">
+     <property name="text">
       <string>Number of &amp;rows:</string>
      </property>
-     <property name="buddy" >
+     <property name="buddy">
       <cstring>m_spRows</cstring>
      </property>
     </widget>
    </item>
-   <item row="1" column="1" >
-    <widget class="QSpinBox" name="m_spRows" >
-     <property name="minimum" >
+   <item row="1" column="1">
+    <widget class="QSpinBox" name="m_spRows">
+     <property name="minimum">
       <number>1</number>
      </property>
     </widget>
    </item>
-   <item row="2" column="0" >
-    <widget class="QLabel" name="label_3" >
-     <property name="text" >
+   <item row="2" column="0">
+    <widget class="QLabel" name="label_3">
+     <property name="text">
       <string>&amp;Spacing:</string>
      </property>
-     <property name="buddy" >
+     <property name="buddy">
       <cstring>m_leSpacing</cstring>
      </property>
     </widget>
    </item>
-   <item row="2" column="1" >
-    <widget class="QLineEdit" name="m_leSpacing" />
+   <item row="2" column="1">
+    <widget class="QLineEdit" name="m_leSpacing"/>
    </item>
   </layout>
  </widget>
- <includes/>
  <resources/>
  <connections/>
 </ui>
diff --git a/src/dialogs/usermenu/usermenudialog_base.ui b/src/dialogs/usermenu/usermenudialog_base.ui
index 7e2a12b8..5191aab3 100644
--- a/src/dialogs/usermenu/usermenudialog_base.ui
+++ b/src/dialogs/usermenu/usermenudialog_base.ui
@@ -7,7 +7,7 @@
     <x>0</x>
     <y>0</y>
     <width>879</width>
-    <height>627</height>
+    <height>645</height>
    </rect>
   </property>
   <property name="sizePolicy">
@@ -616,7 +616,7 @@
              </widget>
             </item>
             <item row="7" column="2">
-             <widget class="KKeySequenceWidget" name="m_keyChooser" native="true">
+             <widget class="KKeySequenceWidget" name="m_keyChooser">
               <property name="whatsThis">
                <string>Choose or clear a shortcut for this action.</string>
               </property>
@@ -725,19 +725,19 @@
   </layout>
  </widget>
  <customwidgets>
-  <customwidget>
-   <class>KKeySequenceWidget</class>
-   <extends>QWidget</extends>
-   <header>kkeysequencewidget.h</header>
-  </customwidget>
   <customwidget>
    <class>KLineEdit</class>
    <extends>QLineEdit</extends>
    <header>klineedit.h</header>
   </customwidget>
+  <customwidget>
+   <class>KKeySequenceWidget</class>
+   <extends>QWidget</extends>
+   <header>kkeysequencewidget.h</header>
+  </customwidget>
   <customwidget>
    <class>KUrlRequester</class>
-   <extends>QFrame</extends>
+   <extends>QWidget</extends>
    <header>kurlrequester.h</header>
    <container>1</container>
   </customwidget>
diff --git a/src/widgets/configcheckerwidget.ui b/src/widgets/configcheckerwidget.ui
index ca01f286..0d5cb5c5 100644
--- a/src/widgets/configcheckerwidget.ui
+++ b/src/widgets/configcheckerwidget.ui
@@ -1,7 +1,8 @@
-<ui version="4.0" >
+<?xml version="1.0" encoding="UTF-8"?>
+<ui version="4.0">
  <class>ConfigCheckerWidget</class>
- <widget class="QWidget" name="ConfigCheckerWidget" >
-  <property name="geometry" >
+ <widget class="QWidget" name="ConfigCheckerWidget">
+  <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
@@ -9,60 +10,60 @@
     <height>605</height>
    </rect>
   </property>
-  <property name="sizePolicy" >
-   <sizepolicy vsizetype="MinimumExpanding" hsizetype="MinimumExpanding" >
+  <property name="sizePolicy">
+   <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
     <horstretch>1</horstretch>
     <verstretch>1</verstretch>
    </sizepolicy>
   </property>
-  <property name="minimumSize" >
+  <property name="minimumSize">
    <size>
     <width>580</width>
     <height>480</height>
    </size>
   </property>
-  <property name="windowTitle" >
+  <property name="windowTitle">
    <string>Performing System Check</string>
   </property>
-  <layout class="QVBoxLayout" >
+  <layout class="QVBoxLayout">
    <item>
-    <widget class="QLabel" name="m_lbChecking" >
-     <property name="text" >
+    <widget class="QLabel" name="m_lbChecking">
+     <property name="text">
       <string>Checking if your TeX system is installed correctly...</string>
      </property>
-     <property name="wordWrap" >
+     <property name="wordWrap">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item>
-    <widget class="QProgressBar" name="m_progress" >
-     <property name="value" >
+    <widget class="QProgressBar" name="m_progress">
+     <property name="value">
       <number>24</number>
      </property>
     </widget>
    </item>
    <item>
-    <widget class="QGroupBox" name="m_grpResults" >
-     <property name="sizePolicy" >
-      <sizepolicy vsizetype="MinimumExpanding" hsizetype="MinimumExpanding" >
+    <widget class="QGroupBox" name="m_grpResults">
+     <property name="sizePolicy">
+      <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
        <horstretch>1</horstretch>
        <verstretch>1</verstretch>
       </sizepolicy>
      </property>
-     <property name="title" >
+     <property name="title">
       <string>Results</string>
      </property>
-     <layout class="QGridLayout" >
-      <item row="0" column="0" >
-       <widget class="QListWidget" name="m_lstResults" >
-        <property name="sizePolicy" >
-         <sizepolicy vsizetype="MinimumExpanding" hsizetype="MinimumExpanding" >
+     <layout class="QGridLayout">
+      <item row="0" column="0">
+       <widget class="QListWidget" name="m_lstResults">
+        <property name="sizePolicy">
+         <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
           <horstretch>1</horstretch>
           <verstretch>1</verstretch>
          </sizepolicy>
         </property>
-        <property name="minimumSize" >
+        <property name="minimumSize">
          <size>
           <width>480</width>
           <height>0</height>
@@ -75,8 +76,7 @@
    </item>
   </layout>
  </widget>
- <layoutdefault spacing="6" margin="11" />
- <pixmapfunction></pixmapfunction>
+ <layoutdefault spacing="6" margin="11"/>
  <resources/>
  <connections/>
 </ui>
diff --git a/src/widgets/graphicsconfigwidget.ui b/src/widgets/graphicsconfigwidget.ui
index cccdae56..e340976a 100644
--- a/src/widgets/graphicsconfigwidget.ui
+++ b/src/widgets/graphicsconfigwidget.ui
@@ -6,7 +6,7 @@
    <rect>
     <x>0</x>
     <y>0</y>
-    <width>563</width>
+    <width>590</width>
     <height>278</height>
    </rect>
   </property>
diff --git a/src/widgets/latexconfigwidget.ui b/src/widgets/latexconfigwidget.ui
index aa8a49d0..4a5e392e 100644
--- a/src/widgets/latexconfigwidget.ui
+++ b/src/widgets/latexconfigwidget.ui
@@ -6,7 +6,7 @@
    <rect>
     <x>0</x>
     <y>0</y>
-    <width>628</width>
+    <width>654</width>
     <height>466</height>
    </rect>
   </property>
@@ -269,16 +269,16 @@
  </widget>
  <layoutdefault spacing="6" margin="11"/>
  <customwidgets>
-  <customwidget>
-   <class>KComboBox</class>
-   <extends>QComboBox</extends>
-   <header>kcombobox.h</header>
-  </customwidget>
   <customwidget>
    <class>KLineEdit</class>
    <extends>QLineEdit</extends>
    <header>klineedit.h</header>
   </customwidget>
+  <customwidget>
+   <class>KComboBox</class>
+   <extends>QComboBox</extends>
+   <header>kcombobox.h</header>
+  </customwidget>
  </customwidgets>
  <includes>
   <include location="global">kconfig.h</include>
diff --git a/src/widgets/latextoolconfigwidget.ui b/src/widgets/latextoolconfigwidget.ui
index 24d4794a..a4351f43 100644
--- a/src/widgets/latextoolconfigwidget.ui
+++ b/src/widgets/latextoolconfigwidget.ui
@@ -1,7 +1,8 @@
-<ui version="4.0" >
+<?xml version="1.0" encoding="UTF-8"?>
+<ui version="4.0">
  <class>LaTeXToolConfigWidget</class>
- <widget class="QWidget" name="LaTeXToolConfigWidget" >
-  <property name="geometry" >
+ <widget class="QWidget" name="LaTeXToolConfigWidget">
+  <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
@@ -9,52 +10,61 @@
     <height>121</height>
    </rect>
   </property>
-  <property name="sizePolicy" >
-   <sizepolicy vsizetype="Fixed" hsizetype="Preferred" >
+  <property name="sizePolicy">
+   <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
-  <layout class="QVBoxLayout" >
-   <property name="margin" >
+  <layout class="QVBoxLayout">
+   <property name="leftMargin">
+    <number>0</number>
+   </property>
+   <property name="topMargin">
+    <number>0</number>
+   </property>
+   <property name="rightMargin">
+    <number>0</number>
+   </property>
+   <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
-    <widget class="QCheckBox" name="m_ckRootDoc" >
-     <property name="toolTip" >
+    <widget class="QCheckBox" name="m_ckRootDoc">
+     <property name="toolTip">
       <string>Check if root document is a LaTeX root before running LaTeX on it</string>
      </property>
-     <property name="text" >
+     <property name="text">
       <string>Check if &amp;root document</string>
      </property>
     </widget>
    </item>
    <item>
-    <widget class="QCheckBox" name="m_ckJump" >
-     <property name="toolTip" >
+    <widget class="QCheckBox" name="m_ckJump">
+     <property name="toolTip">
       <string>Jump to first error in case running LaTeX failed</string>
      </property>
-     <property name="text" >
+     <property name="text">
       <string>&amp;Jump to first error</string>
      </property>
     </widget>
    </item>
    <item>
-    <widget class="QCheckBox" name="m_ckAutoRun" >
-     <property name="toolTip" >
+    <widget class="QCheckBox" name="m_ckAutoRun">
+     <property name="toolTip">
       <string>Automatically run Asymptote, BibTeX, MakeIndex and rerun LaTeX when necessary</string>
      </property>
-     <property name="text" >
+     <property name="text">
       <string>Automatically run additional tools</string>
      </property>
     </widget>
    </item>
    <item>
     <spacer>
-     <property name="orientation" >
+     <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
-     <property name="sizeHint" stdset="0" >
+     <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
@@ -67,7 +77,6 @@
   <zorder>m_ckRootDoc</zorder>
   <zorder>m_ckAutoRun</zorder>
  </widget>
- <pixmapfunction></pixmapfunction>
  <resources/>
  <connections/>
 </ui>
diff --git a/src/widgets/maintoolconfigwidget.ui b/src/widgets/maintoolconfigwidget.ui
index 5b2bf866..c8a590b6 100644
--- a/src/widgets/maintoolconfigwidget.ui
+++ b/src/widgets/maintoolconfigwidget.ui
@@ -680,21 +680,27 @@
      <property name="sizeType">
       <enum>QSizePolicy::MinimumExpanding</enum>
      </property>
+     <property name="sizeHint" stdset="0">
+      <size>
+       <width>0</width>
+       <height>0</height>
+      </size>
+     </property>
     </spacer>
    </item>
   </layout>
  </widget>
  <customwidgets>
-  <customwidget>
-   <class>KComboBox</class>
-   <extends>QComboBox</extends>
-   <header>kcombobox.h</header>
-  </customwidget>
   <customwidget>
    <class>KLineEdit</class>
    <extends>QLineEdit</extends>
    <header>klineedit.h</header>
   </customwidget>
+  <customwidget>
+   <class>KComboBox</class>
+   <extends>QComboBox</extends>
+   <header>kcombobox.h</header>
+  </customwidget>
  </customwidgets>
  <tabstops>
   <tabstop>m_cbConfig</tabstop>
diff --git a/src/widgets/newdocumentwidget.ui b/src/widgets/newdocumentwidget.ui
index 101892c5..50370c6d 100644
--- a/src/widgets/newdocumentwidget.ui
+++ b/src/widgets/newdocumentwidget.ui
@@ -1,45 +1,46 @@
-<ui version="4.0" >
+<?xml version="1.0" encoding="UTF-8"?>
+<ui version="4.0">
  <class>NewDocumentWidget</class>
- <widget class="QWidget" name="NewDocumentWidget" >
-  <property name="geometry" >
+ <widget class="QWidget" name="NewDocumentWidget">
+  <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
-    <width>441</width>
+    <width>476</width>
     <height>217</height>
    </rect>
   </property>
-  <layout class="QVBoxLayout" >
+  <layout class="QVBoxLayout">
    <item>
-    <widget class="QGroupBox" name="groupBox2" >
-     <property name="title" >
+    <widget class="QGroupBox" name="groupBox2">
+     <property name="title">
       <string>Document Type</string>
      </property>
-     <layout class="QVBoxLayout" >
+     <layout class="QVBoxLayout">
       <item>
-       <widget class="QLabel" name="textLabel1" >
-        <property name="text" >
+       <widget class="QLabel" name="textLabel1">
+        <property name="text">
          <string>Please select the type of document you want to create:</string>
         </property>
-        <property name="wordWrap" >
+        <property name="wordWrap">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
-       <layout class="QHBoxLayout" >
+       <layout class="QHBoxLayout">
         <item>
-         <widget class="KComboBox" name="documentTypeComboBox" />
+         <widget class="KComboBox" name="documentTypeComboBox"/>
         </item>
         <item>
          <spacer>
-          <property name="orientation" >
+          <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
-          <property name="sizeType" >
+          <property name="sizeType">
            <enum>QSizePolicy::Expanding</enum>
           </property>
-          <property name="sizeHint" stdset="0" >
+          <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>20</height>
@@ -53,27 +54,27 @@
     </widget>
    </item>
    <item>
-    <widget class="QGroupBox" name="groupBox1" >
-     <property name="title" >
+    <widget class="QGroupBox" name="groupBox1">
+     <property name="title">
       <string>Template</string>
      </property>
-     <layout class="QVBoxLayout" >
+     <layout class="QVBoxLayout">
       <item>
-       <widget class="QLabel" name="textLabel1_2" >
-        <property name="text" >
+       <widget class="QLabel" name="textLabel1_2">
+        <property name="text">
          <string>Please select the template that should be used:</string>
         </property>
-        <property name="wordWrap" >
+        <property name="wordWrap">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
-       <widget class="TemplateIconView" native="1" name="templateIconView" />
+       <widget class="TemplateIconView" name="templateIconView" native="true"/>
       </item>
       <item>
-       <widget class="QCheckBox" name="quickStartWizardCheckBox" >
-        <property name="text" >
+       <widget class="QCheckBox" name="quickStartWizardCheckBox">
+        <property name="text">
          <string>Start the Quick Start wizard when creating an empty LaTeX file</string>
         </property>
        </widget>
@@ -83,8 +84,7 @@
    </item>
   </layout>
  </widget>
- <layoutdefault spacing="6" margin="11" />
- <pixmapfunction></pixmapfunction>
+ <layoutdefault spacing="6" margin="11"/>
  <customwidgets>
   <customwidget>
    <class>KComboBox</class>
diff --git a/src/widgets/processtoolconfigwidget.ui b/src/widgets/processtoolconfigwidget.ui
index 84c9f059..7cdece76 100644
--- a/src/widgets/processtoolconfigwidget.ui
+++ b/src/widgets/processtoolconfigwidget.ui
@@ -104,7 +104,6 @@ p, li { white-space: pre-wrap; }
    </item>
   </layout>
  </widget>
- <pixmapfunction></pixmapfunction>
  <customwidgets>
   <customwidget>
    <class>KLineEdit</class>
diff --git a/src/widgets/quicktoolconfigwidget.ui b/src/widgets/quicktoolconfigwidget.ui
index 0abfa1c5..0ad15780 100644
--- a/src/widgets/quicktoolconfigwidget.ui
+++ b/src/widgets/quicktoolconfigwidget.ui
@@ -7,7 +7,7 @@
     <x>0</x>
     <y>0</y>
     <width>570</width>
-    <height>266</height>
+    <height>274</height>
    </rect>
   </property>
   <property name="sizePolicy">
@@ -160,23 +160,12 @@
   </layout>
  </widget>
  <layoutdefault spacing="6" margin="11"/>
- <pixmapfunction></pixmapfunction>
  <customwidgets>
   <customwidget>
    <class>KComboBox</class>
    <extends>QComboBox</extends>
    <header>kcombobox.h</header>
   </customwidget>
-  <customwidget>
-   <class>QListWidget</class>
-   <extends>QListWidget</extends>
-   <header>klistwidget.h</header>
-  </customwidget>
-  <customwidget>
-   <class>QPushButton</class>
-   <extends>QPushButton</extends>
-   <header>kpushbutton.h</header>
-  </customwidget>
  </customwidgets>
  <resources/>
  <connections/>
diff --git a/src/widgets/scriptingconfigwidget.ui b/src/widgets/scriptingconfigwidget.ui
index 1c828a55..f7b8f94a 100644
--- a/src/widgets/scriptingconfigwidget.ui
+++ b/src/widgets/scriptingconfigwidget.ui
@@ -1,8 +1,9 @@
-<ui version="4.0" >
+<?xml version="1.0" encoding="UTF-8"?>
+<ui version="4.0">
  <author>Michel Ludwig (michel.ludwig@kdemail.net)</author>
  <class>KileWidgetScriptingConfig</class>
- <widget class="QWidget" name="KileWidgetScriptingConfig" >
-  <property name="geometry" >
+ <widget class="QWidget" name="KileWidgetScriptingConfig">
+  <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
@@ -10,96 +11,96 @@
     <height>198</height>
    </rect>
   </property>
-  <property name="sizePolicy" >
-   <sizepolicy vsizetype="Preferred" hsizetype="Preferred" >
+  <property name="sizePolicy">
+   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
-  <property name="windowTitle" >
+  <property name="windowTitle">
    <string>Scripting Support</string>
   </property>
-  <layout class="QVBoxLayout" >
+  <layout class="QVBoxLayout">
    <item>
-    <widget class="QCheckBox" name="kcfg_ScriptingEnabled" >
-     <property name="sizePolicy" >
-      <sizepolicy vsizetype="Minimum" hsizetype="Minimum" >
+    <widget class="QCheckBox" name="kcfg_ScriptingEnabled">
+     <property name="sizePolicy">
+      <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
-     <property name="text" >
+     <property name="text">
       <string>Enable &amp;scripting</string>
      </property>
-     <property name="checked" >
+     <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
-    <widget class="QGroupBox" name="executionTimeLimitGroupBox" >
-     <property name="enabled" >
+    <widget class="QGroupBox" name="executionTimeLimitGroupBox">
+     <property name="enabled">
       <bool>true</bool>
      </property>
-     <property name="sizePolicy" >
-      <sizepolicy vsizetype="Preferred" hsizetype="Expanding" >
+     <property name="sizePolicy">
+      <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
-     <property name="title" >
+     <property name="title">
       <string>Execution Time Limit</string>
      </property>
-     <property name="flat" >
+     <property name="flat">
       <bool>true</bool>
      </property>
-     <layout class="QVBoxLayout" >
+     <layout class="QVBoxLayout">
       <item>
-       <widget class="QCheckBox" name="kcfg_TimeLimitEnabled" >
-        <property name="sizePolicy" >
-         <sizepolicy vsizetype="Minimum" hsizetype="Minimum" >
+       <widget class="QCheckBox" name="kcfg_TimeLimitEnabled">
+        <property name="sizePolicy">
+         <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
-        <property name="text" >
+        <property name="text">
          <string>&amp;Limit the execution time of scripts</string>
         </property>
-        <property name="checked" >
+        <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
-       <layout class="QHBoxLayout" >
+       <layout class="QHBoxLayout">
         <item>
-         <widget class="QLabel" name="textLabel1" >
-          <property name="sizePolicy" >
-           <sizepolicy vsizetype="Preferred" hsizetype="Preferred" >
+         <widget class="QLabel" name="textLabel1">
+          <property name="sizePolicy">
+           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
-          <property name="text" >
+          <property name="text">
            <string>&amp;Time limit (seconds):</string>
           </property>
-          <property name="wordWrap" >
+          <property name="wordWrap">
            <bool>false</bool>
           </property>
-          <property name="buddy" >
+          <property name="buddy">
            <cstring>kcfg_TimeLimit</cstring>
           </property>
          </widget>
         </item>
         <item>
          <spacer>
-          <property name="orientation" >
+          <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
-          <property name="sizeType" >
+          <property name="sizeType">
            <enum>QSizePolicy::Fixed</enum>
           </property>
-          <property name="sizeHint" >
+          <property name="sizeHint" stdset="0">
            <size>
             <width>16</width>
             <height>20</height>
@@ -108,27 +109,27 @@
          </spacer>
         </item>
         <item>
-         <widget class="QSpinBox" name="kcfg_TimeLimit" >
-          <property name="sizePolicy" >
-           <sizepolicy vsizetype="Fixed" hsizetype="Minimum" >
+         <widget class="QSpinBox" name="kcfg_TimeLimit">
+          <property name="sizePolicy">
+           <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
-          <property name="minValue" stdset="0" >
+          <property name="minValue" stdset="0">
            <number>1</number>
           </property>
          </widget>
         </item>
         <item>
          <spacer>
-          <property name="orientation" >
+          <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
-          <property name="sizeType" >
+          <property name="sizeType">
            <enum>QSizePolicy::Expanding</enum>
           </property>
-          <property name="sizeHint" >
+          <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
@@ -143,13 +144,13 @@
    </item>
    <item>
     <spacer>
-     <property name="orientation" >
+     <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
-     <property name="sizeType" >
+     <property name="sizeType">
       <enum>QSizePolicy::Expanding</enum>
      </property>
-     <property name="sizeHint" >
+     <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
@@ -159,8 +160,7 @@
    </item>
   </layout>
  </widget>
- <layoutdefault spacing="6" margin="11" />
- <pixmapfunction></pixmapfunction>
+ <layoutdefault spacing="6" margin="11"/>
  <resources/>
  <connections>
   <connection>
@@ -169,11 +169,11 @@
    <receiver>executionTimeLimitGroupBox</receiver>
    <slot>setEnabled(bool)</slot>
    <hints>
-    <hint type="sourcelabel" >
+    <hint type="sourcelabel">
      <x>20</x>
      <y>20</y>
     </hint>
-    <hint type="destinationlabel" >
+    <hint type="destinationlabel">
      <x>20</x>
      <y>20</y>
     </hint>
@@ -185,11 +185,11 @@
    <receiver>kcfg_TimeLimit</receiver>
    <slot>setEnabled(bool)</slot>
    <hints>
-    <hint type="sourcelabel" >
+    <hint type="sourcelabel">
      <x>20</x>
      <y>20</y>
     </hint>
-    <hint type="destinationlabel" >
+    <hint type="destinationlabel">
      <x>20</x>
      <y>20</y>
     </hint>
@@ -201,11 +201,11 @@
    <receiver>textLabel1</receiver>
    <slot>setEnabled(bool)</slot>
    <hints>
-    <hint type="sourcelabel" >
+    <hint type="sourcelabel">
      <x>20</x>
      <y>20</y>
     </hint>
-    <hint type="destinationlabel" >
+    <hint type="destinationlabel">
      <x>20</x>
      <y>20</y>
     </hint>
diff --git a/src/widgets/structureviewconfigwidget.ui b/src/widgets/structureviewconfigwidget.ui
index 4bcf5013..83f5b9ad 100644
--- a/src/widgets/structureviewconfigwidget.ui
+++ b/src/widgets/structureviewconfigwidget.ui
@@ -6,7 +6,7 @@
    <rect>
     <x>0</x>
     <y>0</y>
-    <width>523</width>
+    <width>544</width>
     <height>373</height>
    </rect>
   </property>
diff --git a/src/widgets/symbolviewconfigwidget.ui b/src/widgets/symbolviewconfigwidget.ui
index fa32f9bb..b0766b54 100644
--- a/src/widgets/symbolviewconfigwidget.ui
+++ b/src/widgets/symbolviewconfigwidget.ui
@@ -6,7 +6,7 @@
    <rect>
     <x>0</x>
     <y>0</y>
-    <width>504</width>
+    <width>525</width>
     <height>227</height>
    </rect>
   </property>
-- 
2.20.1

